crates/holochain/src/conductor/cell.rs

```diff
@@ -760,7 +760,7 @@ impl Cell {
                         .query_row(
                             "SELECT Action.blob as action_blob
                     FROM DhtOp
-                    LEFT JOIN Action ON Action.hash = DhtOp.action_hash
+                    JOIN Action ON Action.hash = DhtOp.action_hash
```

Comment:
Just to double-check my understanding... are all the removals of LEFT throughout this pr because we ended up going with putting the warrants in the same action table?

Reply:
Yes, adding the LEFTs in the previous PR was premature